snapcraft/plugins/catkin.py

```diff
@@ -340,40 +352,41 @@ def __init__(self, ros_distro, ros_package_path, rosdep_path,
         self._rosdep_cache_path = os.path.join(self._rosdep_path, 'cache')
 
     def setup(self):
+        # Make sure we can run multiple times without error, while leaving the
+        # capability to re-initialize, by making sure we clear the sources.
+        shutil.rmtree(self._rosdep_sources_path, ignore_errors=True)
```

Comment:
Ah, good point. I threw that in there to make sure it was okay if the files didn't exist, but I should just be catching that exception. Good catch. Fixing.